src/Components/CategoryFacet.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { headlessEngine } from "../Engine";
+import {
+  CategoryFacet as CategoryFacetType,
+  CategoryFacetState,
+  buildCategoryFacet
+} from "@coveo/headless";
+
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import FormLabel from "@material-ui/core/FormLabel";
+import Box from "@material-ui/core/Box";
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
+
+const hoveredStyle = {
+  cursor: "pointer",
+  color: "primary"
+};
+
+interface ICategoryFacetProps {
+  title: string;
+  field: string;
+  // basePath: [];
```

Comment:
One comment you forgot :)

Reply:
OK, I fixed those two issues and have synced. Thank you!